_includes/side-nav-fast.html

```diff
@@ -0,0 +1,14 @@
+<ul class="doc-side-nav">
+   <li><h5 class="doc-side-nav-title">
+    <a href="{{ site.baseurl }}/fast/">Fast Tips</a></h5>
+   </li>
+   {% assign sorted_posts = site.posts | sort: 'order' %}
+
+   {% for post in sorted_posts %}
+     {% if post.title contains "Performance Tip of the Week" %}
+     <li><h6 class="doc-side-nav-list-item">
+         <a href="{{ post.url }}">{{ post.title | replace: 'Tip of the Week', 'TotW'}}</a></h6>
```

Comment:
Tip -> Tips, similar to above?

Reply:
Same as above. This is a matcher only for a page title, which is singular. The logic here may be a bit brittle (a tip needs to be titled "Performance Tip of the Week...") but that's a separate issue.